config/v1/types_infrastructure.go

```diff
@@ -182,6 +182,25 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// provisioningInterface is the name of the network interface on a Baremetal server connected
+	// to the provisioning network.
+	ProvisioningInterface string `json:"provisioningInterface"`
+
+	// provisioningIp is the VIP to be used by the provisioning service. This must be on the provisioning
+	// subnet, and outside of the DHCP range.
+	ProvisioningIp string `json:"provisioningIp"`
```

Comment:
IP

Reply:
In the related Installer PR we have two install-config entries, `ProvisioniningIP` and `ProvisioningNetworkCIDR`

https://github.com/openshift/installer/pull/2449

Is there a preference in terms of API conventions with whether we pass both `ProvisioniningIP` and `ProvisioningNetworkCIDR` here, or just ProvisioningIP and make that CIDR format, e.g append the subnet?